### ISSUE TITLE
AMP optimizer generate invalid AMP page

### ISSUE THREAD

**Issue by venkatesh-nagineni** [ampproject/amp-toolbox#958]
I try to implement AMP optimizer using nodejs but i got invalidated amp page because it removes all amp extensions like amp-form, amp-list etc.., this is something strange, when i use online playground https://toolbox-optimizer.glitch.me/, would be fine, it gives validated page, Problem appear if i use amp optimizer with nodejs, any idea. Thanks in advance

here it is what i tried

```
const AmpOptimizer = require('@ampproject/toolbox-optimizer');
const fs = require('fs').promises;

const validAmpTransformation = async () => {
    try {
        // create the AMP Optimizer instance
        const ampOptimizer = AmpOptimizer.create();
        const html = await fs.readFile('./amp-template.html', 'utf8');
        const transformedHtml = await ampOptimizer.transformHtml(html);
        await fs.writeFile('./optimizer.html', transformedHtml);
    } catch (error) {
        // send original html instead of any errors during optimization
    }
}

validAmpTransformation();
```

**Comment by venkatesh-nagineni** [comment]
problem with latest version, previous version 2.7.0-alpha.1 working fine, even optimizer-express have dependency with 2.7.0-alpha.1 instead of latest version (2.7.0-alpha.4) https://github.com/ampproject/amp-toolbox/blob/main/packages/optimizer-express/package.json

**Comment by sebastianbenz** [comment]
This should be fixed with the latest release 2.7.1
